### ISSUE TITLE
Incorrect pickaxe upgrade

### ISSUE THREAD

**Issue by Pegwald** [aura-project/aura#482]
A player wanted the silver ore upgrade on their pickaxe, but got instead a way different one. That player now has a chance of getting two ores at once of any kind of ore. Double iron, copper, silver and gold, similar to the gathering knife upgrade which increases your chance of getting two whool by 40%.

On the wiki I can't find a similar upgrade like this for the pickaxe :/

**Comment by Sairii** [comment]
execs hacks got into master then huh?

**Comment by exectails** [comment]
I know we've talked about this issue a while ago, but I can't find it right now... maybe it was on the chat and got lost =| Thanks for making it an issue.

The problem is that originally I only implemented the official bonus "collecting_bonus", because I didn't remember the item specific bonuses. That bonus adds the double drop. However, in combination with "collecting_bonus_product", that bonus somehow becomes a bonus for a higher chance instead, and for that I never had a log.

Basically, we need someone with such an upgrade on an official server to log the item. Logging it is simple, turn on the logger, drop the item, and pick it back up. Then stop logging either post the log or a screen shot of the ItemNew packet here.

**Comment by exectails** [comment]
I got the required data, so I can fix this now. I have a question though, for those who have used these upgrades on officials, do the following chances sound right? Basically, based on the data I have, it seems like the goal of these upgrades is to increase the chance to get the respective ore to ~25%.

![ss 2016-11-19 at 02 46 21](https://cloud.githubusercontent.com/assets/5593920/20455889/0ea4546e-ae67-11e6-92a5-6f44299da7bc.png)

**Comment by exectails** [comment]
@Pegwald To fix the pickaxe on your server, you can run the following SQL query on your database:

```SQL
UPDATE items AS i
INNER JOIN creatures AS cr ON i.creatureId = cr.creatureId
SET meta1 = CONCAT(`meta1`, "CTBONUSPT:4:64006;")
WHERE cr.name = "Insert Character Name" AND i.itemId = 40025 AND i.meta1 LIKE "%CTBONUS%" AND i.meta1 NOT LIKE "%CTBONUSPT%"
```

Just replace "Insert Character Name" on the last line with the actual name, for example:
```SQL
UPDATE items AS i
INNER JOIN creatures AS cr ON i.creatureId = cr.creatureId
SET meta1 = CONCAT(`meta1`, "CTBONUSPT:4:64006;")
WHERE cr.name = "exec" AND i.itemId = 40025 AND i.meta1 LIKE "%CTBONUS%" AND i.meta1 NOT LIKE "%CTBONUSPT%"
```

This adds the silver upgrade bonus to all pickaxes the character has, that have the double bonus, but are missing the silver bonus.
